src/modules/blocks/rsvp/title/styles.pcss

```diff
@@ -0,0 +1,18 @@
+.tribe-editor__ticket__content-row--title {
+	margin-bottom: 20px;
```

Comment:
Same comment about using `--tec-spacer-4` for this one.

Reply:
Thanks! I've applied this fix